src/user-guide/writing-suites.rst

```diff
@@ -62,28 +67,28 @@ A typical example:
 .. code-block:: bash
 
    /path/to/my/suite   # suite configuration directory
-       suite.rc           # THE SUITE CONFIGURATION FILE
+       flow.cylc           # THE SUITE CONFIGURATION FILE
        bin/               # scripts and executables used by tasks
            foo.sh
            bar.sh
            ...
        # (OPTIONAL) any other suite-related files, for example:
-       inc/               # suite.rc include-files
-           nwp-tasks.rc
-           globals.rc
+       inc/               # flow.cylc include-files
+           nwp-tasks.cylc
```

Comment:
are these having their file extensions changing too?

Reply:
Those files are made up examples I think. But I figure there's no point referring to `.rc` files anymore, everything should be `.cylc` from now on?